modules/drivers/bigquery-cloud-sdk/src/metabase/driver/bigquery_cloud_sdk/query_processor.clj

```diff
@@ -984,11 +983,6 @@
   [_driver]
   :mysql)
 
-;; convert LocalDate to an OffsetDateTime in UTC since BigQuery doesn't handle LocalDates as we'd like
-(defmethod driver.sql/->prepared-substitution [:bigquery-cloud-sdk LocalDate]
-  [driver t]
-  (driver.sql/->prepared-substitution driver (t/offset-date-time t (t/local-time 0) (t/zone-offset 0))))
-
```

Comment:
Yeah, the comment is not super clear on why. I assume it could be due to potentially comparing date against the DateTime columns. As the other tests are passing I _believe_ this is not the case anymore.